intervalometer.c

```diff
@@ -189,6 +205,8 @@ static struct flipvalo_trigger* flipvalo_get_trigger(
             return &canon_ir_trigger;
         case FvTrigNikon:
             return &nikon_ir_trigger;
+	case FvTrigPentax:
```

Comment:
```suggestion
        case FvTrigPentax:
```
Looks like your editor placed a tab here. Should be 8 spaces.